definitions/features/downstream.rb

```diff
@@ -30,6 +30,16 @@ def setup_repositories(version)
     end
   end
 
+  def absent_repos(version)
+    all_repos = execute!(
+      "subscription-manager repos --list | grep 'Repo ID' | " \
+         "awk -F':' '/:/{gsub(/ /, \"\", $2); print $2}'"
```

Comment:
No need to use `grep` ;) It can be written like:

```
subscription-manager repos --list | awk -F':' '/Repo ID/{gsub(/ /, "", $2); print $2}

Reply:
Ah. Nice! thanks @sean797